keyboards/kippersun/nanetta65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,84 @@
+		KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_LGUI, KC_LALT, KC_MENU, KC_LCTL, KC_SPC),
+
+	[1]=LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	[2]=LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	[3]=LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+
+};
+
+
+led_config_t g_led_config = {
+    {
+        // Key Matrix to LED Index
+        //假设你的键盘上3行3列，就在这里写上3行3列的NO_LED。注意最后一行末尾没有逗号
+        {NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED},
+        {NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED},
+        {NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED},
+				{NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED},
+			  {NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED,NO_LED, NO_LED, NO_LED}
+    },
+    {
+        // LED Index to Physical Position
+        //LED的坐标，你只有一个灯所以坐标没必要，就0,0好了
+	{0,0}
+    },
+    {
+        // LED Index to Flag
+        //用于给灯分组，一个灯就不用分组了
+        1
+    }
+};
+
```

Comment:
This should also be removed/replacing when using [`RGBLIGHT`](https://docs.qmk.fm/#/feature_rgblight).